libsolidity/formal/CHC.h

```diff
@@ -246,7 +255,13 @@ class CHC: public SMTEncoder
 	/// @returns a predicate application after checking the predicate's type.
 	smtutil::Expression predicate(Predicate const& _block);
 	/// @returns the summary predicate for the called function.
-	smtutil::Expression predicate(FunctionCall const& _funCall);
+	smtutil::Expression predicate(
```

Comment:
Wouldn't it make sense to keep the overload that takes just `FunctionCall`?

Reply:
Done